### PR TITLE
Remove config capability

### PR DESCRIPTION
**Pull request by tigrannajaryan**
This existed also in OpenCensus but as far as I know was never
used in real world. Removing now as unneeded functionality.
If there is a future evidence that this is needed we can add back.

### CODE REVIEW

File: opentelemetry/proto/trace/v1/trace_config.proto [hunk 1]
```diff
@@ -1,77 +0,0 @@
...
-
-package opentelemetry.proto.trace.v1;
-
-option java_multiple_files = true;
-option java_package = "io.opentelemetry.proto.trace.v1";
-option java_outer_classname = "TraceConfigProto";
-
-// Global configuration of the trace service. All fields must be specified, or
-// the default (zero) values will be used for each type.
-message TraceConfig {
```
Comment (bogdandrutu) [inline opentelemetry/proto/trace/v1/trace_config.proto]: Please don't remove this for the moment. We still have a config capability for the library. I know it is manual for the moment but we can use this as reference to what can be configured.

Comment (tigrannajaryan) [reply opentelemetry/proto/trace/v1/trace_config.proto]: Do you suggest to remove `rpc Config` from `TraceService` but keep trace_config.proto file for future work?

Comment (bogdandrutu) [reply opentelemetry/proto/trace/v1/trace_config.proto]: Yes

Comment (tigrannajaryan) [reply opentelemetry/proto/trace/v1/trace_config.proto]: Done

**Pull request merged**